src/components/features/ProfileMenu.tsx

```diff
@@ -35,46 +37,53 @@ export default function ProfileMenu() {
   }
 
   const onConfirmClick = () => {
+    // Log out from the context only!
+    setUser(null)
```

Comment:
we dont need this since its already cleared out inside the logout function

Reply:
fixed